projects/smart-alarm.md

```diff
@@ -7,4 +7,8 @@ Users often set their alarm clocks expecting their morning routine and commute t
 We propose to create an alarm app that will periodically check your morning commute and estimate the amount of time your commute will take and wake you up with enough time to make your appointments.
 
 ###Team
-Edward Wu, Gideon Glass
+[Edward Wu](people/edward-wu.md), [Gideon Glass](people/gideon-glass.md)
```

Comment:
These links don't work, they should be prefixed with a slash or they are relative.